Automattic/jetpack
# VideoPress: Move jetpack_videopress REST custom field to VideoPress package.

This PR moves the `jetpack_videopress` REST custom field to VideoPress package from the Jetpack plugin.
Having this functionality in the VideoPress package will guarantee the `jetpack_videopress` field will be present in media endpoint responses when either Jetpack or VideoPress plugin is active.

Similar work has already been done in #26043 for the `jetpack_videopress_guid` field

#### Changes proposed in this Pull Request:

* VideoPress: Introduce `WPCOM_REST_API_V2_Attachment_VideoPress_Data` for registering the `jetpack_videopress` field
* Jetpack: Remove the corresponding file from the Jetpack plugin

#### Jetpack product discussion


pe4Cmx-fL-p2#comment-128

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Test with a site with some VideoPress videos and the Jetpack plugin active and connected
* Hit the `wp/v2/media` endpoint 
* Confirm you see the `jetpack_videopress` in the request-response body only for VideoPress posts.
* Deactivate the Jetpack plugin and activate the VideoPress plugin
* Hit the media endpoint again and confirm the above

<img width="871" alt="Screen Shot 2022-09-05 at 17 55 50" src="https://user-images.githubusercontent.com/77539/188511753-688caea0-f387-487c-8f78-9dd0b7b1eeb2.png">
